styles/widgets/generic/color-schemes/light/generic.light.less

```diff
@@ -1224,401 +1005,206 @@
 */
-@treelist-spin-icon-color: lighten(@base-icon-color, 38.5%);
+@datagrid-search-color: @base-inverted-text-color;
+
 /**
-* @name 152. Selection spin icon color
+* @name 125. Spin icon color
 * @type color
-* @group treelist
+* @group datagrid
 */
-@treelist-selection-spin-icon-color: @treelist-spin-icon-color;
+@datagrid-spin-icon-color: lighten(@base-icon-color, 38.5%);
 
 /**
-* @name 160. Highlighted text background color
+* @name 110. Highlighted text background color
 * @type color
-* @group treelist
+* @group datagrid
 * @paletteColorOpacity 1
 */
-@treelist-search-bg: @base-accent;
+@datagrid-search-bg: @base-accent;
 
 /**
-* @name 170. Error row text color
+* @name 115. Error row text color
 * @type color
-* @group treelist
+* @group datagrid
 */
-@treelist-row-error-color: @base-inverted-text-color;
+@datagrid-row-error-color: @base-inverted-text-color;
 
 /**
-* @name 180. Error row background color
+* @name 120. Error row background color
 * @type color
-* @group treelist
+* @group datagrid
 */
-@treelist-row-error-bg: mix(@base-invalid-color, @treelist-base-background-color, 60%);
+@datagrid-row-error-bg: mix(@base-invalid-color, @datagrid-base-background-color, 60%);
+@datagrid-link-color: @base-link-color;
 
 /**
-* @name 40. Link color
+* @name 75. Focused cell border color
 * @type color
-* @group treelist
+* @group datagrid
 * @paletteColorOpacity 1
 */
-@treelist-link-color: @base-link-color;
+@datagrid-focused-border-color: @base-focus-bg;
+/**
+* @name 80. Editor background
+* @type color
+* @group datagrid
+*/
+@datagrid-editor-bg: @datagrid-base-background-color;
+
+@datagrid-drop-highlight-bg: @base-accent;
+@datagrid-drop-highlight-color: @base-inverted-text-color;
+@datagrid-columnchooser-bg: @base-element-bg;
 
 /**
-* @name 130. Focused cell border color
+* @name 85. Row alternation background color
 * @type color
-* @group treelist
-* @paletteColorOpacity 1
+* @group datagrid
 */
-@treelist-focused-border-color: @base-focus-color;
+@datagrid-row-alternation-bg: darken(@datagrid-base-background-color, 4%);
+@datagrid-columnchooser-item-bg: @datagrid-columnchooser-bg;
+
 /**
-* @name 131. Editor background
+* @name 100. Summary item color
 * @type color
-* @group treelist
+* @group datagrid
 */
-@treelist-editor-bg: @treelist-base-background-color;
+@datagrid-summary-color: fade(@datagrid-base-color, 70%);
+@datagrid-columnchooser-message-color: lighten(@datagrid-base-color, 40%);
+@datagrid-summary-bg: @datagrid-base-background-color;
+
+@datagrid-overlay-content-shadow-color: fade(@base-shadow-color, 15%);
+@datagrid-columnchooser-shadow-color: fade(@base-shadow-color, 20%);
+
+@datagrid-columnchooser-overlay-border-radius: @base-border-radius-large; // TODO: may be not need
+
+@datagrid-column-separator-bg: fade(@base-accent, 50%);
+@datagrid-text-stub-background-image-path: url(images/widgets/generic/color-schemes/light/grid/text-stub.png);
+
+// TreeList
+
+@treelist-base-color: @base-text-color;
```

Comment:
Why is not possible to customize Tree List styles?

Reply:
When Tree List was implemented we decided that it should have separate settings in theme builder.

Reply:
It is the designer's decision because they had the same settings.